File: src/main/java/com/zunza/buythedip/auth/oauth2/exception/SocialEmailAlreadyRegisteredException.java
package com.zunza.buythedip.auth.oauth2.exception;

import org.springframework.security.core.AuthenticationException;

import com.zunza.buythedip.common.CustomException;

public class SocialEmailAlreadyRegisteredException extends AuthenticationException {
	private static final String MESSAGE = "이메일이 기존 계정과 연결되어 있습니다.";

	public SocialEmailAlreadyRegisteredException() {
		super(MESSAGE);
	}
}

File: src/main/kotlin/com/zunza/buythedip/auth/oauth2/exception/SocialEmailAlreadyRegisteredException.kt
package com.zunza.buythedip.auth.oauth2.exception

import org.springframework.security.core.AuthenticationException

class SocialEmailAlreadyRegisteredException() : AuthenticationException(MESSAGE) {
    companion object {
        private const val MESSAGE = "이메일이 기존 계정과 연결되어 있습니다."
    }
}
